src/functions-and-arrays.js

```diff
@@ -121,15 +121,30 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(arr) {} 
+//wrong! ???
+/* let values = []
 
+  wordsUnique.forEach(item) => {
+    if(!wordsUnique.includes(item))
+        values.push(item)
+}
+values = values.concat(wordsUnique);
+
+return values.sort ()
+}
+*/
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+function doesWordExist(word) {
```

Comment:
So we know that this `function` will accept two `arguments` the first one will be the `array` of words, and the other one will be the specific `word` that you want to check if it exists in the array. So what you will need to do is, create a `loop` that will run over the `array` and in which `element` it will check if `word` is equal to the current `element` of the array:

```
function doesWordExist(array, word) {
  if (array.length === 0) {
    return null;
  }
  for (let i = 0; i < array.length; i++) {
    if (array[i] === word) {
      return true;
    }
  }
  return false;
}
```